src/abi.h

```diff
@@ -327,4 +327,43 @@ struct netplan_net_definition {
     /* netplan-feature: eswitch-mode */
     char* embedded_switch_mode;
     gboolean sriov_delay_virtual_functions_rebind;
+
+    /* vxlan */
+    guint vxlan_vni;
+    NetplanNetDefinition* vxlan_link;
+    char* vxlan_bridge;
+    GArray* vxlans;
+    gboolean neigh_suppress;
+
+    /* vrf */
```

Comment:
```suggestion
    /* vrf */
    /* netplan-feature: vrf */
```

Let's add a feature-flag for VRF, too

Reply:
Done.